test/test_file_storage.cpp

```diff
@@ -52,6 +52,7 @@ void setup_test_storage(file_storage& st)
 	TEST_EQUAL(st.file_offset(file_index_t{3}), 60000);
 
 	TEST_EQUAL(st.total_size(), 100000);
+	TEST_EQUAL(st.size_on_disk(), 100000);
```

Comment:
For what purpose make another variable, if it will also be equal to total_size

Reply:
As I've already explained, the total size and size_on_disk are the same for torrents with no pad files.

```
total_size == size_on_disk + pad_file_size
```

Reply:
@arvidn Maybe I'm not seeing something. Where do you filter .pad files?

Reply:
@arvidn Okay. I found where the pad files are filtered. But you have to test my .torrent file. The information in it doesn't match. Maybe there are some other problems.